### PR TITLE
Icon: Calendar and Chart icons added

### PR DESCRIPTION
**Pull request by tumentumurchudur**
- Added calendar and chart icons in the Icon component

screen shot 2016-01-04 at 9 55 36 am

### CODE REVIEW

File: src/components/Icon.js [hunk 1]
```diff
@@ -102,6 +102,16 @@ class Icon extends React.Component {
               l-68.3-68.3c-5.9-5.9-15.6-5.9-21.7,0c-5.9,5.9-5.9,15.6,0,21.7l68.3,68.3l-68.3,68.3C212.7,330.3,212.7,340,218.6,346z'/>
           </g>
         );
+      case 'calendar':
+        return (
+          <path d="M412.4,123.4h-86.5v-17.3c0-6.2-5-11.2-11.2-11.2c-6.2,0-11.2,5-11.2,11.2v17.3h-94.8v-17.3
```
Comment (cerinman) [inline src/components/Icon.js]: Change the double quotes to single quotes.

File: src/components/Icon.js [hunk 2]
```diff
@@ -141,6 +151,19 @@ class Icon extends React.Component {
...
+            <path d="M424,120.2c0-6.2-5-11.2-11.2-11.2H100.2C94,109,89,114,89,120.2v270.7c0,6.2,5,11.2,11.2,11.2
+              h312.7c6.2,0,11.2-5,11.2-11.2V120.2z M401,380H111V132h290V380z"/>
+            <path d="M142.7,347.3c14.2,0,25.7-11.5,25.7-25.7c0-3.1-0.5-6-1.5-8.8l40.1-41.2c2.6,0.9,5.3,1.3,8.2,1.3
+              c3.7,0,7.2-0.8,10.3-2.2l37.3,31.8c-0.6,2.1-0.8,4.3-0.8,6.5c0,14.2,11.5,25.7,25.7,25.7c14.2,0,25.7-11.5,25.7-25.7
+              c0-5-1.5-9.7-4-13.7l47.9-76.3c0.9,0.1,1.9,0.2,2.8,0.2c14.2,0,25.7-11.5,25.7-25.7c0-14.2-11.5-25.7-25.7-25.7
+              c-14.2,0-25.7,11.5-25.7,25.7c0,5,1.5,9.7,4,13.7l-47.9,76.3c-0.9-0.1-1.9-0.2-2.8-0.2c-3.7,0-7.2,0.8-10.4,2.2L240,253.8
+              c0.6-2.1,0.8-4.3,0.8-6.5c0-14.2-11.5-25.7-25.7-25.7c-14.2,0-25.7,11.5-25.7,25.7c0,3.1,0.5,6.1,1.6,8.8l-40.1,41.1
+              c-2.6-0.9-5.3-1.3-8.2-1.3c-14.2,0-25.7,11.5-25.7,25.7C117,335.8,128.5,347.3,142.7,347.3z"/>
+          </g>
+        );
```
Comment (bsbeeks) [inline src/components/Icon.js]: Double quotes should be converted to single quotes.

File: src/components/Icon.js [hunk 3]
```diff
@@ -141,6 +151,19 @@ class Icon extends React.Component {
           <path d='M256,200.2c3.6,0,7.1,1.4,9.9,4.1l83.7,83.7c5.4,5.4,5.4,14.3,0,19.7c-5.4,5.4-14.3,5.4-19.7,0L256,233.9
             l-73.8,73.8c-5.4,5.4-14.3,5.4-19.7,0c-5.4-5.4-5.4-14.3,0-19.7l83.7-83.7C248.9,201.6,252.4,200.2,256,200.2z' />
         );
+      case 'chart':
+        return (
+          <g>
+            <path d="M424,120.2c0-6.2-5-11.2-11.2-11.2H100.2C94,109,89,114,89,120.2v270.7c0,6.2,5,11.2,11.2,11.2
```
Comment (cerinman) [inline src/components/Icon.js]: same here

File: src/components/Icon.js [hunk 4]
```diff
@@ -141,6 +151,19 @@ class Icon extends React.Component {
...
+              h312.7c6.2,0,11.2-5,11.2-11.2V120.2z M401,380H111V132h290V380z"/>
+            <path d="M142.7,347.3c14.2,0,25.7-11.5,25.7-25.7c0-3.1-0.5-6-1.5-8.8l40.1-41.2c2.6,0.9,5.3,1.3,8.2,1.3
```
Comment (cerinman) [inline src/components/Icon.js]: and here

**Comment by tumentumurchudur** [comment]
Yes, changed the double quote to single. All checks passing now.

**Comment by cerinman** [comment]
:shipit:

**Pull request merged**